Resources/public/js/views/fields/ez-relationlist-editview.js

```diff
@@ -132,17 +132,33 @@ YUI.add('ez-relationlist-editview', function (Y) {
                 relatedContentsJSON = [];
 
             Y.Array.each(relatedContents, function (value) {
```

Comment:
You can do it more performant and proper way (if it was in ES6 it would be even shorter, but the unit testing setup doesn't allow ES6):
```javascript
relatedContentsJSON = relatedContentsJSON.reduce(function (total, value) {
    var relatedContentJSON = value.toJSON();

    if (this._isNewRelation(value) || relatedContentJSON.resources.MainLocation) {
        total.push(relatedContentJSON);
    }

    return total;
}.bind(this), []);
```

Reply:
@sunpietro It seems that unit tests are failing on the native`bind` method, so we decided to back to `Y.bind` (probably might be missing support for it in PhantomJS <=2.1).

Reply:
Ahh, yes. This is so limiting ...